src/modules/billing/routes/invoices.js

```diff
@@ -23,3 +23,21 @@ exports.patchInvoice = {
     }
   }
 };
+
+exports.resetIsFlaggedForRebilling = {
+  method: 'PATCH',
+  path: '/water/1.0/billing/invoices/rebillingflag/{batchId}/org/{originalInvoiceId}/rebill/{rebillInvoiceId}',
```

Comment:
the route here receives the batch id but the controller doesn't use it so this can be removed. i would suggest a route of: 
`/water/1.0/billing/invoices/{invoiceId}/reset-rebilling-flag`

the ui will then send the original invoice id to this route to set the status to rebill when the status is rebilled or null if the status is rebill